frontends/infinite-corridor/src/pages/SearchPage.tsx

```diff
@@ -92,51 +107,66 @@ const SearchPage: React.FC = () => {
     )
 
   return (
-    <div>
-      <BannerPageWrapper>
```

Comment:
I swapped this out for [`BannerPage`](https://github.com/mitodl/open-discussions/blob/3dfa43edf54606f96b0a69a9c2aaa02b9e237f32/frontends/ol-util/src/components/PageBanner.tsx#L130), which does most of this automatically.